multihash.go

```diff
@@ -267,9 +263,8 @@ func Decode(buf []byte) (*DecodedMultihash, error) {
 // Encode a hash digest along with the specified function code.
 // Note: the length is derived from the length of the digest itself.
 func Encode(buf []byte, code uint64) ([]byte, error) {
-	if !ValidCode(code) {
-		return nil, ErrUnknownCode
-	}
+	// REVIEW: if we remove the strict ValidCode check, this can no longer error.  Change signiture?
```

Comment:
I'd leave it. It's slightly annoying but it means we have the error response if we need it and changing this would be more annoying.